dts/arm/st/u5/stm32u5_usbotg_hs.dtsi

```diff
@@ -27,6 +27,8 @@
 		compatible = "st,stm32u5-otghs-phy";
 		clocks = <&rcc STM32_CLOCK(AHB2, 15)>,
 			 <&rcc STM32_SRC_HSE OTGHS_SEL(0)>;
+		clock-reference = "SYSCFG_OTG_HS_PHY_CLK_16MHz";
+		status = "disabled";
```

Comment:
Could you place `status = "disabled"` after `#phy-cells` property? Prefer have the `status` property always last among node properties.